### DISCUSSION TITLE
when adding new panel fill the available empty in dashboard

### DISCUSSION THREAD

**Discussion by LikhithaSanga11** [react-grid-layout/react-grid-layout#1586] [Q&A]
When I am adding a new panel I want to fill the empty space first, but when I am adding it is not filling the empty space and adding panel at bottom

**Comment by LikhithaSanga11** [comment]
@STRML can you please help me regarding this issue. Thank you in advance

**Comment by Kakomis** [comment]
Hi, Could you solve it?
